crates/s3s/src/http/body.rs

```diff
@@ -52,7 +54,7 @@ impl Body {
         }
     }
 
```

Comment:
Should we add a variant for `http_body_util::BoxBody` instead of going threw `DynStream` 

we could also just change `Body` to be a wrapper over `http_body_util::BoxBody` like Axum

Reply:
A wrapper over `http_body_util::BoxBody` like Axum is better. But it needs a refactor of the stream mechanism (also related to #136), which should be seperated from this PR.

A variant for `http_body_util::BoxBody` and using `DynStream` are both OK for now. You can try `BoxBody` if you wish.